consensus/module.go

```diff
@@ -201,3 +201,7 @@ func (m *consensusModule) handleHotstuffMessage(msg *typesCons.HotstuffMessage)
 	// Note that the leader also acts as a replica, but this logic is implemented in the underlying code.
 	leaderHandlers[msg.Step](m, msg)
 }
+
+func (m *consensusModule) GetBlockHeight() uint64 {
```

Comment:
Any chance we could use the `GetLatestBlockHeight` from persistence instead?

Reply:
Like discussed over discord, not until we have that exposed somehow. I've left a comment in `shared/modules/consensus_module.go` to keep track of this as you suggested.